main/util/src/mill/util/Jvm.scala

```diff
@@ -454,6 +627,44 @@ object Jvm extends CoursierSupport {
     )(new Ctx.Home { override def home = os.home })
   }
 
+  def createClassLoader(
```

Comment:
Let's add a short Scaladoc here describing what it does, with docs for the various parameters